.gitmodules

```diff
@@ -0,0 +1,5 @@
+[submodule "media_viewing_widgets"]
```

Comment:
Could be that this line has to be changed into : [submodule "taplt/media_viewing_widgets"]